README.rst

```diff
@@ -1,6 +1,6 @@
 python-socketio
 ===============
-
+.. image:: https://cdn-1.chadfreeman.repl.co/static/python-socket.IO-no-background.png
```

Comment:
if you change -no-background.png to .jpg you get an image with a white background but who would want that